server/embed/config.go

```diff
@@ -254,6 +254,9 @@ type Config struct {
 	// SocketOpts are socket options passed to listener config.
 	SocketOpts transport.SocketOpts `json:"socket-options"`
 
+	// GRPCInterceptor specifies the grpc server interceptors.
+	GRPCInterceptor grpc.UnaryServerInterceptor
```

Comment:
How about export ability to add customized `grpc.ServerOption`. like grpcCustomOptions []gprc.ServerOption

Reply:
sgtm, but i find hard to dynamic change `grpc.ServerOption` to `grpc.UnaryServerInterceptor`
`impossible type switch case: opt (type grpc.ServerOption) cannot have dynamic type grpc.UnaryServerInterceptor (missing grpc.apply method)`

could you give me some advice for this ? @nolouch